packages/ui/package.json

```diff
@@ -36,7 +36,7 @@
   "scripts": {
     "start": "start-storybook -p 6006 -s public",
     "build": "build-storybook -s public",
-    "build:js": "tsc -p tsconfig.build.json",
+    "typecheck": "tsc -p tsconfig.build.json",
```

Comment:
add this extra one `"typecheck": "tsc"` and keep the `build:js` as it was. One is to build the package and one to do the typechecking.

Reply:
I did it, but when I ran the `"typecheck": "tsc"` the compiler created a lot of .js files besides the .ts files, I thought we wanted to avoid that. Let me try again.